includes/api/ApiQueryAllUsers.php

```diff
@@ -0,0 +1,383 @@
+<?php
+/**
+ *
+ *
+ * Created on July 7, 2007
+ *
+ * Copyright © 2007 Yuri Astrakhan <Firstname><Lastname>@gmail.com
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ * http://www.gnu.org/copyleft/gpl.html
+ *
+ * @file
+ */
+use Wikia\Util\PerformanceProfilers\UsernameUseProfiler;
+
+/**
+ * Query module to enumerate all registered users.
+ *
+ * @ingroup API
+ */
+class ApiQueryAllUsers extends ApiQueryBase {
+	public function __construct( $query, $moduleName ) {
+		parent::__construct( $query, $moduleName, 'au' );
+	}
+
+	public function execute() {
+		$usernameUseProfiler = new UsernameUseProfiler( __CLASS__, __METHOD__ );
+		$db = $this->getDB();
+		$params = $this->extractRequestParams();
+
+		$prop = $params['prop'];
+		if ( !is_null( $prop ) ) {
+			$prop = array_flip( $prop );
+			$fld_blockinfo = isset( $prop['blockinfo'] );
+			$fld_editcount = isset( $prop['editcount'] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "editcount" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)